symphony/lib/toolkit/class.frontendpage.php

```diff
@@ -536,10 +536,9 @@ private function __buildPage()
         $xml->prependChild($params);
 
         $this->setXML($xml);
-        $xsl = '<?xml version="1.0" encoding="UTF-8"?>
-        <xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
-            <xsl:import href="' . str_replace('\\', '/', $page['filelocation']) . '"/>
-        </xsl:stylesheet>';
+        $xsl = '<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">' .
+               '    <xsl:import href="' . str_replace('\\', '/', str_replace(' ', '%20', $page['filelocation'])) . '"/>' .
```

Comment:
This can be simplified further as `str_replace(array('\\', ' '), array('/', '%20'), $page['filelocation'])` will save invoking the function twice.